Flow.Launcher.Infrastructure/Constant.cs

```diff
@@ -19,7 +19,7 @@ public static class Constant
         public static readonly string RootDirectory = Directory.GetParent(ApplicationDirectory).ToString();
         
         public static readonly string PreinstalledDirectory = Path.Combine(ProgramDirectory, Plugins);
-        public const string Issue = "https://github.com/Flow-Launcher/Flow.Launcher/issues/new/choose";
+        public const string Issue = "https://github.com/Flow-Launcher/Flow.Launcher/issues";
```

Comment:
This change is making the link point to entirely different section, is this correct? Isnt this constant used for leading user to submit a new issue?

Reply:
Just updated variable and method names around the GitHub issues url to make it more readable.